main.js

```diff
@@ -12,4 +13,211 @@ closeMenu.addEventListener('click', () => {
+    demoLink: 'https://github.com/Ghilain/mobile-version-skeleton/',
+    spec: 'd-none1',
+    butt: 'd-none',
+    ulist: 'li-bt',
+  },
+  {
+    id: 5,
+    name: 'Profesional Art Printing Data',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    featuredImage: './Images/Snapshoot-page-2.png',
+    technologies: 'html',
+    technologies1: 'css',
+    technologies2: 'javascript',
+    sourceLink: 'https://ghilain.github.io/mobile-version-skeleton/',
+    demoLink: 'https://github.com/Ghilain/mobile-version-skeleton/',
+    spec: 'd-none1',
+    butt: 'd-none',
+    ulist: 'li-bt',
+  },
+  {
+    id: 6,
+    name: 'Profesional Art Printing Data',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    featuredImage: './Images/Snapshoot-page-2.png',
+    technologies: 'html',
+    technologies1: 'css',
+    technologies2: 'javascript',
+    sourceLink: 'https://ghilain.github.io/mobile-version-skeleton/',
+    demoLink: 'https://github.com/Ghilain/mobile-version-skeleton/',
+    spec: 'd-none1',
+    butt: 'd-none',
+    ulist: 'li-bt',
+  },
+];
+
+const projectCards = projectsList.map((cards) => `
+<li class="size bg${cards.id}">
+          <div class="temp2 ${cards.spec}">
+            <div class="pad">
+              <h3 class="work card-heading">${cards.name}</h3>
+              <p class="text text-2">
+               ${cards.description}
+              </p>
+              <ul class="tech tech-2 ${cards.ulist} ">
+              <li>${cards.technologies}</li>
+              <li>${cards.technologies1}</li>
+              <li>${cards.technologies2}</li>
+              </ul>
+            </div>
+          </div>
+          <div>
+            <button id="btncli${cards.id}" type="submit" class="btn btns btn-2 ${cards.butt}">see project</button>
+          </div>
+        </li>
+
+`).join('');
+
+cardList.innerHTML += projectCards;
+
+// Popup window
+const popupWindow = document.querySelector('.bgpopup');
+
+const popWindowContent = [
+  {
+    name1: 'Multi Post Stories',
+    name2: 'Keeping track of hundreds  of components website',
+    closee: 'X',
+    li1: 'html',
+    li2: 'bootstrap',
+    li3: 'Ruby on rails',
+    featuredImagepop: './images/popupdesk.png',
+    popDescription: `Lorem Ipsum is simply dummy text of the printing and typesetting
+  industry. Lorem Ipsum has been the industry's standard dummy text
+  ever since the 1500s, when an unknown printer took a galley of
+  type and scrambled it to make a type specimen book. It has
+  survived not only five centuries, but also the leap into
+  electronic typesetting, remaining essent`,
+    imageSeelive: 'images/seelive.png',
+    imageSeeproject: 'images/logogit.png',
+
+  },
+];
+
+const popupWindowBox = popWindowContent.map((popwindow) => `
```

Comment:
- [ ] Please make sure to make a dynamic pop window each project should have its own popup window data in your version if we added new data to the array it will break the code, in your version all the projects have one popup with the same data please fix it, the word dynamic means that the code is capable of change depending on the data it is given.